exec/exec.go

```diff
@@ -105,18 +107,39 @@ func Output(ctx context.Context, command ...string) ([]byte, []byte, error) {
 	}
 
 	cmd := exec.CommandContext(ctx, command[0], arg...)
-
-	var outbuf, errbuf bytes.Buffer
 	cmd.Stdout = &outbuf
 	cmd.Stderr = &errbuf
 
 	if err := cmd.Start(); err != nil {
-		return nil, nil, errors.Wrapf(err, "unable to run command %s", cmd)
+		return nil, nil, 0, errors.Wrapf(err, "unable to run command %s", cmd)
 	}
 
-	if err := cmd.Wait(); err != nil {
-		return outbuf.Bytes(), errbuf.Bytes(), err
+	err = cmd.Wait()
```

Comment:
Oh, sorry, I didn't realize that err was an argument to exitCode(err). my bad...